README.md

```diff
@@ -50,6 +61,7 @@ Many Windows developers have found that setting up an Ubuntu virtual machine is
 1. `cd code-dot-org/aws`
 1. `bundle install`
 1. `cd ..`
+1. <code>sudo chown \`whoami\` ../.npm</code>
```

Comment:
I'll update the line; I believe that one of either the bundle or rake tasks runs a `sudo npm install`